xtest-serialization/tests/serialize.rs

```diff
@@ -218,3 +220,83 @@ fn serial_many_dim_ron() {
         assert_eq!(a, a_de);
     }
 }
+
+#[test]
+fn serial_ixdyn_borsh() {
+    {
+        let a = arr0::<f32>(2.72).into_dyn();
+        let serial = borsh::to_vec(&a).unwrap();
+        println!("Serde encode {:?} => {:?}", a, serial);
+        let res = borsh::from_slice::<ArcArray<f32, _>>(&serial);
+        println!("{:?}", res);
+        assert_eq!(a, res.unwrap());
+    }
+
+    {
+        let a = arr1::<f32>(&[2.72, 1., 2.]).into_dyn();
+        let serial = borsh::to_vec(&a).unwrap();
+        println!("Serde encode {:?} => {:?}", a, serial);
+        let res = borsh::from_slice::<ArrayD<f32>>(&serial);
+        println!("{:?}", res);
+        assert_eq!(a, res.unwrap());
+    }
+
+    {
+        let a = arr2(&[[3., 1., 2.2], [3.1, 4., 7.]])
+            .into_shape(IxDyn(&[3, 1, 1, 1, 2, 1]))
+            .unwrap();
+        let serial = borsh::to_vec(&a).unwrap();
+        println!("Serde encode {:?} => {:?}", a, serial);
+        let res = borsh::from_slice::<ArrayD<f32>>(&serial);
+        println!("{:?}", res);
+        assert_eq!(a, res.unwrap());
+    }
+}
+
+#[test]
+fn serial_many_dim_borsh() {
+    use borsh::from_slice as borsh_deserialize;
+    use borsh::to_vec as borsh_serialize;
+
+    {
+        let a = arr0::<f32>(2.72);
+
+        let a_s = borsh_serialize(&a).unwrap();
+
+        let a_de: ArcArray<f32, _> = borsh_deserialize(&a_s).unwrap();
+
+        assert_eq!(a, a_de);
```

Comment:
I know that you respected the style of the previous block (serde), but please don't use double interline. Those 4 lines can use 4 lines.

Reply:
Right, this is just copy+paste artifact.... I'll try to fix this on Monday.